solidstart-tanstackquery-tailwind-modules/src/components/UserRepos/UserRepos.tsx

```diff
@@ -1,10 +1,16 @@
 import { For, Show, splitProps } from 'solid-js';
 import { RepoCard } from '~/components/RepoCard';
 import { RepoFilter } from '~/components/RepoFilter';
+import { Pagination } from '../Pagination';
 import { RepoInfos } from '../ProfilePage/ProfilePage';
 
 const UserRepos = (props: RepoInfos) => {
-  const [local] = splitProps(props, ['repos', 'pageInfo', 'languages']);
+  const [local] = splitProps(props, [
```

Comment:
I actually don't understand why we use `splitProps` here at all, is there something I am missing?

Reply:
So solidjs provide a safe way to pick/destruct some props value while keeping its reactivity, but most times we don't need it because we are not destructing props.

Reply:
But it is also not needed here